inst/templates/get-started.Rmd

```diff
@@ -383,6 +383,31 @@ vroom::vroom_write(to_edit, "to_edit.csv", delim = ",")
 writexl::write_xlsx(to_edit, "to_edit.xlsx")
 ```
 
+### Calculation of coverage statistics
+
+```{r}
+# matched companies based on suggested matches
+matched <- nrow(to_edit %>% dplyr::filter(suggest_match == TRUE))
+
+# share of matched companies from total loanbook companies
+matched_share <- nrow(to_edit %>% dplyr::filter(suggest_match == TRUE)) / nrow(loanbook %>% distinct(company_name))
+
+# Number and share of matched companies classified by misc_info from total loanbook companies
+# misc_info should be replaced with variables like sectors, headcount etc.
+share_df <- (loanbook %>% dplyr::count(misc_info)) %>%
+  dplyr::left_join(to_edit %>%
+    dplyr::filter(suggest_match == TRUE) %>%
+    dplyr::count(misc_info), by = c("misc_info"), suffix = c("_loanbook", "_merged"))
+share_df$share_misc <- share_df$n_merged / share_df$n_loanbook
+
+# Guidance on loan exposure
+# misc_info and id_tilt should be replaced based on target column names
+exposure <- to_edit %>%
+  dplyr::filter(suggest_match == TRUE) %>%
+  dplyr::group_by(misc_info) %>%
+  dplyr::summarise(sum_exposure = sum(id_tilt))
```

Comment:
`count()` wraps `group_by() + summarize( ... sum(..` so you may be able to simplify and clarify this code with something like:

```r
exposure <- to_edit %>% 
  filter(suggest_match) %>%
  count(id_tilt)

exposure
```

Reply:
@maurolepore I want to take sum of loan amounts categorised based on misc_info (like sectors etc.) after the matching is done. I tested that count could not wrap both group_by and summarise function for my usecase. Please let me know if I a wrong about this! Thanks!